server/index.js

```diff
@@ -1,12 +1,29 @@
+// server flow: index.js -> routes -> controller -> model -> db-mysql
 const express = require('express');
 const bodyParser = require('body-parser');
 const morgan = require('morgan');
 const path = require('path');
-const db = require('../db-mysql');
+
 const middleware = require('./middleware.js');
+const router = require('./routes.js');
 
 const app = express();
 const PORT = 3000;
+app.set('port', PORT);
+
+// app.use(bodyParser.json());
+// app.use(morgan('dev'));
+app.use(express.json());
+app.use(middleware.httpRequestLogger);
+
+app.get('/api', (req, res) => {
+  console.log('hello inside get route');
+  res.send('response sent correctly!');
+});
+
+app.use('/api/models', router);
```

Comment:
I don't know if `models` should be in the url. From the perspective of an API user who wants to access the reviews, typing in models doesn't make sense.